create_graphics.py

```diff
@@ -546,26 +552,36 @@ def graphics_driver(cla):
             print()
             print((('-' * 80)+'\n') * 2)
 
-            if cla.graphic_type == 'skewts':
-                create_skewt(cla, fhr, grib_path, workdir)
-            else:
-                gribfiles = gather_gribfiles(cla, fhr, gribfiles)
-                create_maps(cla,
-                            fhr=fhr,
-                            gribfiles=gribfiles,
-                            workdir=workdir,
-                            )
-
-            # Zip png files and remove the originals in a subprocess
             if cla.zip_dir:
-                for tile, zipf in zipfiles.items():
-                    png_files = glob.glob(os.path.join(workdir, f'*_{tile}_*{fhr:02d}.png'))
```

Comment:
With the change I mentioned above about using a skewt as a tile, you'd want to change this line:

```
png_files = glob.glob(os.path.join(workdir, f'*{tile}_*{fhr:02d}.png'))
```

Since skewt png files are named like this: `skewt_BAO_99999_f06.png`